packages/react-auth/src/index.tsx

```diff
@@ -1,8 +1,14 @@
 import React, { createContext, ReactNode, useContext, useEffect, useState } from 'react'
 
+import type { User } from '@nhost/hasura-auth-js'
 import { NhostClient } from '@nhost/nhost-js'
 
-export const AuthContext = createContext({
+type NhostContext = {
```

Comment:
I think we could export this type to make it available to users of the library.

One scenario where this could come particularly useful is when they try to create their own context which would extend our `AuthContext` in some way.

The same is also true for our `NhostAuthProvider` which could have its props exported for the same reason.

Reply:
I'm leaving this here for future reference.